pallets/capacity/src/migration/v3.rs

```diff
@@ -126,7 +126,7 @@ where
 			return Ok(Vec::new())
 		}
 
-		let pallet_prefix = StakingAccountLedger::<T>::module_prefix();
+		let pallet_prefix = StakingAccountLedger::<T>::pallet_prefix();
```

Comment:
nit: This migration has already run, but is this a bug or a change in the api to use pallet instead of module?

Reply:
Change in api. It wont compile.